test/integration/camera.cc

```diff
@@ -293,5 +296,134 @@ TEST_P(CameraSensorTest, LInt8ImagesWithBuiltinSDF)
   ImageFormatLInt8LInt16(GetParam());
 }
 
+//////////////////////////////////////////////////
+void CameraSensorTest::CameraIntrinsics(const std::string &_renderEngine)
+{
+  // Get the darn test data
```

Comment:
I realize this comment was copied from other tests above, but it seems a bit silly. I would remove it.

Reply:
Done https://github.com/gazebosim/gz-sensors/pull/256/commits/76c86dbc1a3351bd929e4440f47c201ce986158b